### DISCUSSION TITLE
CANNOT apply student developer kit

### DISCUSSION THREAD

**Discussion by sigar27** [community/community#40796] [GitHub Education]
When I applied for the student discount, I selected the school email address and pressed the Continue button, but the button was disabled and the website did not respond.

**Comment by ossanna16** [comment]
Hi there @sigar27 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by samus-aran** [comment]
Hi @sigar27 👋 

We're happy you're here. 

We apologise you had experienced issues on GitHub Global Campus platform. Are you able to provide more details or screenshots of the issue you are seeing.
